### PR TITLE
Add processing and example e2e of hackathon goal

### PR DESCRIPTION
**Pull request by eedeleon**
I have a cache of the other PR for now, I can rebase on top of the other after it is checked in. Please review the approach to ensure it can be replicated in azureml.explain.model

### CODE REVIEW

File: python/interpret_community/processing.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    if hasattr(base_class, "explain_local"):
+        if not hasattr(base_class, "_explain_local"):
+            base_class._explain_local = base_class.explain_local
+
+        def new_explain_local(self, data, *args, **kwargs):
+            import pdb; pdb.set_trace()
+            if self._preprocessors:
+                for preprocessor in self._preprocessors:
+                    data = preprocessor(data)
+            import pdb; pdb.set_trace()
```
Comment (wamartin-aml) [inline python/interpret_community/processing.py]: remove debugging

Comment (wamartin-aml) [reply python/interpret_community/processing.py]: here and above

File: test/hackathon_results.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
+from sklearn.pipeline import Pipeline
```
Comment (wamartin-aml) [inline test/hackathon_results.py]: We'll need to clean this file up some before checkin - I can work on that when I have some free cycles

**Comment by imatiach-msft** [comment]
closing old PRs

**Pull request closed without merging**